spec/models/request_spec.rb

```diff
@@ -289,8 +289,8 @@ class node_myvm {
         expect(@git_stub.git).to have_received(:commit_all).with('Add myvm')
       end
 
-      it 'returns a success message' do
-        expect(request.push_to_git).to eq(notice: 'Added file and pushed to git.')
+      it 'returns no error message' do
```

Comment:
![image](https://user-images.githubusercontent.com/1346493/52205036-1f6d0080-2876-11e9-9f51-bc787e679212.png)
Regarding the issue of double notifications: I was trying to clarify to the user, that there was an error when associating the users to the VM, but the VM was still created
Do you think it makes sense to remove the success message then, because I think it would be just as confusing the other way around...

Reply:
Is the Creation/Update successfull, when some parameters could not be set correctly?

Reply:
Maybe I will replace the success notification with a warning notification, if there was an error with git

Reply:
sounds good to me!